torchvision/transforms/functional_tensor.py

```diff
@@ -937,8 +937,7 @@ def normalize(tensor: Tensor, mean: List[float], std: List[float], inplace: bool
         mean = mean.view(-1, 1, 1)
     if std.ndim == 1:
         std = std.view(-1, 1, 1)
-    tensor.sub_(mean).div_(std)
-    return tensor
```

Comment:
Behavior is equivalent for plain tensors, but gives us the unwrapping behavior we want from this change. With this in place, explicit unwrapping in `normalize` is no longer needed. Let me know if I should keep it there. Runtime is the same, because here we would implicitly unwrap inside `__torch_function__`.

Reply:
Initially I thought this was a styling change. Then I realize you do this to avoid returning `tensor` (which might be an image) and instead return the unwrapped `torch.Tensor`. Might worth adding a short comment in a follow up PR.